IntegrationTests.Common/IntegrationTests.Common.csproj

```diff
@@ -1,175 +1,38 @@
-    <Compile Include="App_Packages\Remotion.JetBrains.Annotations.Sources.1.15.23.0\UsedImplicitlyAttribute.cs" />
-    <Compile Include="App_Packages\Remotion.Utilities.ArgumentUtility.Sources.1.15.23.0\ArgumentUtility.cs" />
-    <Compile Include="App_Packages\Remotion.Utilities.Assertion.Sources.1.15.23.0\Assertion.cs" />
-    <Compile Include="App_Packages\Remotion.Utilities.NullableTypeUtility.Sources.1.15.23.0\NullableTypeUtility.cs" />
-    <Compile Include="Database\DatabaseConfiguration.cs" />
-    <Compile Include="Database\NorthwindSetup.cs" />
-    <None Include="..\LGPL.licenseheader">
-      <Link>LGPL.licenseheader</Link>
-    </None>
-    <None Include="..\remotion.snk">
-      <Link>remotion.snk</Link>
-    </None>
-    <Compile Include="TestDomain\Northwind\CustomTransformers\DateDiffDayMethodCallTransformer.cs" />
-    <Compile Include="TestDomain\Northwind\CustomTransformers\UserDefinedFunctionTransformer.cs" />
-    <Compile Include="TestDomain\Northwind\SystemTestDataContextExtension.cs" />
-    <Compile Include="TestMode.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="AbstractTestBase.cs" />
-    <Compile Include="TestDomain\Northwind\INorthwindDataProvider.cs" />
-    <Compile Include="TestDomain\Northwind\LinqToSqlNorthwindDataProvider.cs" />
-    <Compile Include="TestDomain\Northwind\NorthwindDataContext.cs" />
-    <Compile Include="TestDomain\Northwind\NorthwindConnectionManager.cs" />
-    <Compile Include="TestDomain\Northwind\RelinqNorthwindDataProvider.cs" />
-    <Compile Include="UnitTests\Utilities\ComparisonResultTest.cs" />
-    <Compile Include="Utilities\BatchCommand.cs" />
-    <Compile Include="Utilities\CheckingTestExecutor.cs" />
-    <Compile Include="Utilities\DatabaseAgent.cs" />
-    <Compile Include="Utilities\SavingTestExecutor.cs" />
-    <Compile Include="TestDomain\SerializerTestClassWithFields.cs" />
-    <Compile Include="TestDomain\SerializerTestClassWithProperties.cs" />
-    <Compile Include="TestDomain\SerializerTestClassWithReferenceTypes.cs" />
-    <Compile Include="UnitTests\Utilities\TestResultSerializerTest.cs" />
-    <Compile Include="Utilities\ComparisonResult.cs" />
-    <Compile Include="Utilities\ITestExecutor.cs" />
-    <Compile Include="Utilities\SqlBatchCommandException.cs" />
-    <Compile Include="Utilities\TestResultSerializer.cs" />
-    <None Include="App_Packages\Remotion.Utilities.Assertion.Sources.1.15.23.0\SharedSource Apache-2.0.licenseheader" />
-    <None Include="App_Packages\Remotion.Utilities.NullableTypeUtility.Sources.1.15.23.0\SharedSource Apache-2.0.licenseheader" />
-    <None Include="packages.config" />
+    <PackageReference Include="Remotion.Linq" Version="2.0.0" />
   </ItemGroup>
+
   <ItemGroup>
-    <None Include="App.config" />
+    <Reference Include="System.Configuration" />
+    <Reference Include="System.Data.Linq" />
```

Comment:
@a-ctor  can we inhert the {{Ssystem.Data.Linq}} property from LinqToSqlAdapter instead of declaring it in every project? and should we?

Reply:
Sadly, this does not work for references. I think we should keep it like this since there is not much added benefit of trying to centralize the reference.